src/Controller/StanfordEarth2PangeaController.php

```diff
@@ -15,7 +15,7 @@ class StanfordEarth2PangeaController extends ControllerBase {
    */
   public function content() {
 
-    $url = 'https://pangea.stanford.edu'.$_SERVER['REQUEST_URI'];
+    $url = 'https://pangea.stanford.edu'. $_SERVER['REQUEST_URI'];
```

Comment:
@ksharp-drupal,

REQUEST_URI is not a safe server variable and should be sanitized to avoid XSS attacks.